image_provisioner/playbooks/roles/docker-config/tasks/main.yaml

```diff
@@ -26,3 +31,4 @@
 
 - name: docker version override
   shell: yum --disablerepo=* localinstall -y http://{{ rhel_repo_hostname }}/brewroot/packages/docker/1.10.3/50.el7/x86_64/docker-1.10.3-50.el7.x86_64.rpm http://{{ rhel_repo_hostname }}/brewroot/packages/docker/1.10.3/50.el7/x86_64/docker-common-1.10.3-50.el7.x86_64.rpm http://{{ rhel_repo_hostname }}/brewroot/packages/docker/1.10.3/50.el7/x86_64/docker-forward-journald-1.10.3-50.el7.x86_64.rpm http://{{ rhel_repo_hostname }}/brewroot/packages/docker/1.10.3/50.el7/x86_64/docker-rhel-push-plugin-1.10.3-50.el7.x86_64.rpm http://{{ rhel_repo_hostname }}/brewroot/packages/docker/1.10.3/50.el7/x86_64/docker-selinux-1.10.3-50.el7.x86_64.rpm http://{{ rhel_repo_hostname }}/brewroot/packages/docker/1.10.3/50.el7/x86_64/oci-register-machine-1.10.3-50.el7.x86_64.rpm http://{{ rhel_repo_hostname }}/brewroot/packages/docker/1.10.3/50.el7/x86_64/oci-systemd-hook-1.10.3-50.el7.x86_64.rpm http://{{ rhel_repo_hostname }}/brewroot/packages/selinux-policy/3.13.1/92.el7/noarch/selinux-policy-3.13.1-92.el7.noarch.rpm http://{{ rhel_repo_hostname }}/brewroot/packages/selinux-policy/3.13.1/92.el7/noarch/selinux-policy-targeted-3.13.1-92.el7.noarch.rpm http://{{ rhel_repo_hostname }}/brewroot/packages/policycoreutils/2.5/5.1.el7/x86_64/policycoreutils-2.5-5.1.el7.x86_64.rpm http://{{ rhel_repo_hostname }}/brewroot/packages/libsemanage/2.5/3.el7/x86_64/libsemanage-2.5-3.el7.x86_64.rpm http://{{ rhel_repo_hostname }}/brewroot/packages/policycoreutils/2.5/5.1.el7/x86_64/policycoreutils-python-2.5-5.1.el7.x86_64.rpm http://{{ rhel_repo_hostname }}/brewroot/packages/setools/3.3.8/1.1.el7/x86_64/setools-libs-3.3.8-1.1.el7.x86_64.rpm http://{{ rhel_repo_hostname }}/brewroot/packages/libsepol/2.5/4.el7/x86_64/libsepol-2.5-4.el7.x86_64.rpm http://{{ rhel_repo_hostname }}/brewroot/packages/libsemanage/2.5/3.el7/x86_64/libsemanage-python-2.5-3.el7.x86_64.rpm http://{{ rhel_repo_hostname }}/brewroot/packages/libselinux/2.5/4.el7/x86_64/libselinux-2.5-4.el7.x86_64.rpm http://{{ rhel_repo_hostname }}/brewroot/packages/libselinux/2.5/4.el7/x86_64/libselinux-utils-2.5-4.el7.x86_64.rpm http://{{ rhel_repo_hostname }}/brewroot/packages/systemd/219/24.el7/x86_64/systemd-219-24.el7.x86_64.rpm http://{{ rhel_repo_hostname }}/brewroot/packages/systemd/219/24.el7/x86_64/systemd-libs-219-24.el7.x86_64.rpm http://{{ rhel_repo_hostname }}/brewroot/packages/systemd/219/24.el7/x86_64/systemd-sysv-219-24.el7.x86_64.rpm http://{{ rhel_repo_hostname }}/brewroot/packages/systemd/219/24.el7/x86_64/libgudev1-219-24.el7.x86_64.rpm http://{{ rhel_repo_hostname }}/brewroot/packages/libselinux/2.5/4.el7/x86_64/libselinux-python-2.5-4.el7.x86_64.rpm http://{{ rhel_repo_hostname }}/brewroot/packages/iptables/1.4.21/17.el7/x86_64/iptables-services-1.4.21-17.el7.x86_64.rpm
```

Comment:
For some context, these overrides are only necessary because of current point of the development cycle we're in in RHEL-land.  We eventually should get away from this because static versions are a bear to manage...basically every time we build an image we're going to have to update this step (and I've been doing this manually).